R/format.R

```diff
@@ -31,7 +44,6 @@
 xportr_format <- function(.df,
                           metadata = NULL,
                           domain = NULL,
```

Comment:
Shouldn't there be a `xportr.format_verbose` parameter?

```suggestion
                          domain = NULL,
                          verbose = getOption("xportr.format_verbose", "none"),
```